core/src/main/java/me/deadlight/ezchestshop/data/ShopContainer.java

```diff
@@ -446,7 +447,14 @@ private static boolean ifHasMoney(OfflinePlayer player, double price) {
         }
     }
 
-    private static void getandgive(OfflinePlayer withdraw, double price, OfflinePlayer deposit) {
+    private static void getandgive(OfflinePlayer withdraw, double price, OfflinePlayer deposit, double taxesvalue) {
+        if(Config.taxesshopplayer != 0){
+            double taxes = price*(taxesvalue)/100;
```

Comment:
Please format your code according to the rest of the plugin.
This means e.g. spaces after `,` statements like `if` or before a `{` or use spaces to separate math expressions - e.g. `price * taxesvalue / 100`. (The `()` around taxesvalue were also not needed.

Furthermore, it's common in java to spell words in camelCase - so e.g. `taxesvalue` becomes `taxesValue`. This improves the readability of your code.